astropy/utils/iers/tests/test_iers.py

```diff
@@ -168,15 +168,17 @@ def test_simple(self):
         predictive_mjd = dat.meta['predictive_mjd']
         dat._time_now = Time(predictive_mjd, format='mjd') + 7 * u.d
 
-        # Look at times before and after the test file begins.
-        assert np.allclose(dat.ut1_utc(Time(50000, format='mjd').jd).value, 0.0262719)
+        # Look at times before and after the test file begins.  0.1292905 is
+        # the IERS-B value from MJD=57359.  The value in
```

Comment:
This is testing a time well before the first value in the test IERS-A file, which it does by assuming that first value for all previous times.  Since the bundled IERS-B overlaps with the test IERS-A, the numeric value will come from the IERS-B table at MJD=59359.0, not the test IERS-A.